elastic/kibana
# [ResponseOps][RuleRegistry] alerts indices broken if Kibana startup pauses for > 20m

## Summary

We saw a stateful project which had a pause of 46m from when Kibana started, to when it completed. It appears ES must have been inaccessible during that time and the rule registry was not able to install the alerting resources. This PR uses an observable from Core to wait to install the alerting resources until ES is ready.


### Checklist



### To verify

- Start Kibana without starting ES
```
yarn start --logging.root.level=debug
```
- Verify that the alerting resources are not installed, you should not see this debug log `Initializing resources for AlertsService`
- Start ES and verify that the resources are installed, and you see the log above
